web/src/components/debug.tsx

```diff
@@ -23,5 +23,3 @@ const DEBUG: FC<DebugProps> = ({ values }) => {
     </div>
   );
 };
-
-export default DEBUG;
```

Comment:
Curious why switch from a default export for a file with a single export?

Reply:
> Curious why switch from a default export for a file with a single export?

Due to react fast refresh check:
```
warning  Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components
```

More info on: https://github.com/ArnaudBarre/eslint-plugin-react-refresh#readme 

Mainly these:
```
The plugin rely on naming conventions (i.e. use PascalCase for components, camelCase for util functions). This is why there are some limitations:

export * are not supported and will be reported as an error
Anonymous function are not supported (i.e export default function() {})
Class components are not supported
All-uppercase function export is considered an error when not using direct named export (ex const CMS = () => <></>; export { CMS })
```